apps/backoffice-v2/src/Router/Router.tsx

```diff
@@ -110,13 +111,25 @@ const router = createBrowserRouter([
                 ],
               },
               {
-                path: '/:locale/statistics',
-                element: <Statistics />,
-                errorElement: <RouteError />,
-              },
-              {
-                path: '/:locale/workflows',
-                element: <Workflows />,
+                path: '/:locale',
+                element: (
+                  <div className={`flex flex-col p-4`}>
+                    <Welcome />
+                    <Outlet />
+                  </div>
+                ),
```

Comment:
1. Rename "Welcome" to "Home"
2. Home should already include the div container and the outlet component.